src/components/Footer.jsx

```diff
@@ -1,111 +1,94 @@
-              </ul>
-            </div>
-          </Col>
-        </Row>
-        <Row>
-          <Col xs={12}>
-            <div className='site-footer-small-print'>
-              <ul className='small-print-nav'>
-                {config.showAccessibilityInfo && (
-                  <li>
-                    <Link to={{ path: '/accessibility' }}>
-                      <FormattedMessage id='accessibilityLink' />
-                    </Link>
-                  </li>
-                )}
-                <li>
-                  <a href={urls.dataProtection} target='_blank' rel='noopener noreferrer'>
-                    <FormattedMessage id='dataProtection' />
-                  </a>
-                </li>
-                {config.enableCookies && (
-                  <li>
-                    <Link to={{ path: '/cookies' }}>
-                      <FormattedMessage id='cookieManagementLink' />
-                    </Link>
-                  </li>
-                )}
-                <li>
-                  <a href={getFeedbackUrl(language)} target='_blank' rel='noopener noreferrer'>
-                    <FormattedMessage id='feedbackLinkText' />
-                  </a>
-                </li>
-                <li>
-                  {getCurrentYear()} <FormattedMessage id='copyrightText' />
-                </li>
-              </ul>
-            </div>
-          </Col>
-        </Row>
-      </div>
-    </footer>
+    <HDSFooter
+      className={classNames(['footer', userIsAdmin && 'footer--is-admin'])}
+      title='Kerrokantasi'
+      logoLanguage={language}
+      korosType={settings.footer.korosType}
+      theme={{
+        '--footer-background': 'var(--color-black)',
+        '--footer-color': 'var(--color-white)',
+      }}
+    >
+      <HDSFooter.Navigation>
+        <HDSFooter.Link label={<FormattedMessage id='hearingsHeaderText' />} href='/hearings/list' />
+        <HDSFooter.Link label={<FormattedMessage id='hearingMapHeaderText' />} href='/hearings/map' />
+        {userIsAdmin && <HDSFooter.Link label={<FormattedMessage id='ownHearings' />} href='/user-hearings' />}
+        {user && <HDSFooter.Link label={<FormattedMessage id='userInfo' />} href='/user-profile' />}
+      </HDSFooter.Navigation>
+      <HDSFooter.Utilities>
+        <HDSFooter.Link
+          label={<FormattedMessage id='feedbackPrompt' />}
+          href={getFeedbackEmailUrl(language)}
+          target='_blank'
+          rel='noopener noreferrer'
+        />
+        <HDSFooter.Link
+          label={<FormattedMessage id='feedbackLinkText' />}
+          href={getFeedbackUrl(language)}
+          target='_blank'
+          rel='noopener noreferrer'
+        />
+      </HDSFooter.Utilities>
+      <HDSFooter.Base
+        copyrightHolder={<FormattedMessage id='copyrightHolder' />}
+        copyrightText={<FormattedMessage id='copyrightText' />}
+        backToTopLabel={<FormattedMessage id='scrollToTop' />}
+        logo={
+          <Logo
```

Comment:
This needs to be added to settings.json aswell to support Turku not using Helsinki logo.

Reply:
Nevermind, it works as requested...